### DISCUSSION TITLE
A new category for ip detection services

### DISCUSSION THREAD

**Discussion by pulsarice** [v2fly/domain-list-community#1807] [General] [answered]
I want to create a category for ip detection services like "ifconfig.me".
What should I name it?
Is it ok to create a "category-ipinfo" file under data directory?

**Comment by MkQtS** [comment, marked as answer]
Thanks to @TripleA150, there is `category-ip-geo-detect` now. https://github.com/v2fly/domain-list-community/commit/809d338494f7c57d4baa6cdb620c583abe74eb5c
